### PR TITLE
Update README to include setup instructions

### LINKED ISSUE #18: Update the readme to show how to set up the project
The readme should be well documented showing all steps on how to set up the project/application

### PR DESCRIPTION
**Pull request by blakes24**
### What does this PR do?
- This PR adds instructions for setting up the app to the README document

### CODE REVIEW

**Review by alexkayabula — CHANGES_REQUESTED** [review]
@blakes24 add a section that details how to run tests.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+
+ - `DATABASE_URL=postgres:///database_name`
+ - `SECRET_KEY=yourSecretKeyHere`
+
+- Do the following in the Terminal:
+
+ - `python3 -m venv venv`
+ - `source venv/bin/activate`
+ - `pip install -r requirements.txt`
+ - `createdb database_name`
```
Comment (alexkayabula) [inline README.md]: what does this line do ? Is it a command we run to create a database or Its an instruction to create a database ?

Comment (blakes24) [reply README.md]: It is a command you run to create a database with the name you give the database in place of database_name. How should I phrase it?

Comment (alexkayabula) [reply README.md]: Do you run that command in the general terminal or it is a `psql` command ?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+
+- Create a .env file in the main directory that contains:
+	- `DATABASE_URL=postgres:///your_database_name`
+	- `SECRET_KEY=yourSecretKeyHere`
+
+- Do the following in the Terminal:
+	- `python3 -m venv venv`
+	- `source venv/bin/activate`
+	- `pip install -r requirements.txt`
+	- `createdb <your_database_name>`
```
Comment (blakes24) [inline README.md:22]: @alexkayabula is this clearer? Is there a better way to phrase it?

File: README.md (hunk 1 above)
Comment (blakes24) [reply README.md]: in the general terminal

File: README.md (hunk 2 above)
Comment (alexkayabula) [reply README.md:22]: @blakes24 what I want you to clarify is whether we run that command on the general terminal or the psql terminal.

**Review by alexkayabula — APPROVED** [review]
LGTM

**Pull request merged**